css/style.css

```diff
@@ -541,6 +543,10 @@ figcaption details p {
     background-color: #000000;
 }
 
+.floe-footer .floe-content a{
+    padding : 0px 3.5px ;
+}
+
```

Comment:
Please leave a space between selectors and the brace "{" for example `a {`.

Also seems we are using em values for padding not px. Please correct throughout.